api/middleware/middleware.go

```diff
@@ -37,6 +37,9 @@ func getOrg(req *http.Request, multiTenant bool) (int, error) {
 	}
 	orgStr := req.Header.Get("x-org-id")
 	if orgStr == "" {
+		orgStr = req.Header.Get("x-grafana-org-id")
+	}
+	if orgStr == "" {
```

Comment:
x-grafana-org-id is just going to be the internal id of the org within the grafana instance, it won't have any correlation with the grafana.com org id